third_party/kubernetes-drain/drain.go

```diff
@@ -357,7 +357,6 @@ func waitForDelete(params waitForDeleteParams) ([]corev1.Pod, error) {
 			default:
 				return false, nil
 			}
-			return false, nil
```

Comment:
This seems like an unrelated change?

Reply:
It's actually both. It's unrelated and `third_party/` is excluded from linters, but I did run my linter on it and well, for the sake of clean code I removed that line. The same change is live [here](https://github.com/kubernetes/kubectl/blob/master/pkg/drain/drain.go#L381-L388)

Reply:
Since the code is changed upstream, could you also update the referenced hash in `third_party/kubernetes-drain/README.md` as well (and verify any other changes that might exist in the newer version)?

Reply:
Done. No changes in the other files.